src/restoreTerminals.ts

```diff
@@ -63,8 +68,15 @@ export default async function restoreTerminals(configuration: Configuration) {
         );
       }
     } else {
+      let color = TerminalColor.white;
+
+      if (terminalWindow.splitTerminals[0]?.color) {
+        color = TerminalColor[terminalWindow.splitTerminals[0].color];
+      }
```

Comment:
You need to check that `terminalWindow.splitTerminals[0]?.color` is an actual color and not something random the user has inputed. Otherwise color will be undefined.

Reply:
In any case, it falls back to white